src/components/ContentOpenWith/IntegrationPortalContainer.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @flow
+ * @file integration portal container
+ * @author Box
+ */
+
+import * as React from 'react';
+import IntegrationPortal from './IntegrationPortal';
+import messages from '../messages';
+import ErrorMask from 'box-react-ui/lib/components/error-mask/ErrorMask';
+import LoadingIndicator from 'box-react-ui/lib/components/loading-indicator/LoadingIndicator';
+import { FormattedMessage } from 'react-intl';
+import uniqueid from 'lodash/uniqueId';
+
+type Props = {
+    hasError: boolean,
+    integrationWindow: any,
+};
+
+const IntegrationPortalContainer = ({ hasError, integrationWindow }: Props) => {
+    return (
+        <IntegrationPortal
+            integrationWindow={integrationWindow}
+            key={uniqueid('integration_portal_')}
```

Comment:
key for?

Reply:
For some reason the portal doesn't unmount when I close the new tab, so the key forces a remount each time.